src/service/routes/push.ts

```diff
@@ -33,15 +35,15 @@ router.get('/:id', async (req, res) => {
   }
 });
 
-router.post('/:id/reject', async (req, res) => {
+router.post('/:id/reject', async (req: Request, res: Response) => {
   if (req.user) {
     const id = req.params.id;
 
     // Get the push request
     const push = await db.getPush(id);
 
     // Get the committer of the push via their email
-    const committerEmail = push.userEmail;
+    const committerEmail = push?.userEmail;
```

Comment:
If an email is missing we shouldn't spend time looking for the email in the DB.
A similar 401 to below would be good